ocs_ci/deployment/helpers/tnf_helpers.py

```
@@ -0,0 +1,772 @@
+    requires ROTA=0.
+
+    Args:
+        node_name (str): Node name
+        device_path (str): Device path (e.g. /dev/vda, /dev/sdb)
+    """
+    import os
+
+    dev_name = os.path.basename(device_path)
+    rotational_path = f"/sys/block/{dev_name}/queue/rotational"
+    try:
+        result = exec_cmd(
```

Ruff (0.16.1)

[error] Function call with `shell=True` parameter identified, security issue

(S604)